### PR TITLE
add scvi recipes

### PR DESCRIPTION
**Pull request by imyiningliu**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [x] This PR does something else (explain below).

### CODE REVIEW

File: recipes/scvi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "adf53e5ce8510adf5d26b5ea4e7f4ed46ac92977a7359303c87727b000a6cb54" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
```
[the remark this answers is not in the document]
Comment (jeff-regier) [reply recipes/scvi/meta.yaml]: Probably want something here like
```
source:
  git_url: https://github.com/YosefLab/scVI.git
  git_rev: 0.1.2
```

[the remark this answers is not in the document]
Comment (imyiningliu) [reply recipes/scvi/meta.yaml]: Thanks Jeff! Seems like the line of `fn` isn't required. I'll remove and submit another PR. If it doesn't work, I'll try using `git`.

**Comment by jeff-regier** [comment]
Huzzah!

**Comment by jeff-regier** [comment]
Thanks for your help, @bgruening !

**Comment by bgruening** [comment]
Your very welcome!

**Pull request merged**